packages/lesswrong/server/resolvers/analyticsResolvers.ts

```diff
@@ -177,27 +195,23 @@ addGraphQLResolvers({
       // keys, the partial is no longer partial
       return postAnalytics as PostAnalyticsResult;
     },
+    // TODO make this handle posts as well
```

Comment:
I'm assuming this TODO is meant to still be here

Reply:
Yeah it is, I'll reuse this for the post analytics page